LEAF_Request_Portal/db_mysql.php

```diff
@@ -290,4 +291,23 @@ public function enableDryRun()
     {
         $this->dryRun = true;
     }
+
+    private function checkLastModified() {
+        //get the last build time
+        $lastBuildTime =  getenv('LAST_BUILD_DATE', true) ?: getenv('LAST_BUILD_DATE');
```

Comment:
Sorry, Greg, I messed you up.  You probably haven't pulled the prototype in a while.  These need to be two different values.  I originally had them the same because I wasn't sure where I wanted to put the default variable.  ;-)  The line construct it to make sure the variable is filled with something usable that won't break things further in the code.

Reply:
Changed code to add default Unix epoch date if LAST_BUILD_DATE not found